06-reactjs/components/movie-form.js

```diff
@@ -0,0 +1,32 @@
+var React = require('react');
+
+var MovieForm = React.createClass({
+    /*That clears the form fields when the form is submitted with valid input.*/
```

Comment:
try to use the code block generate when typing /** + enter, also leave 1 space between function definition and before and after createClass closing and opening brackets